spark/v3.0/spark/src/test/java/org/apache/iceberg/spark/SparkTestBase.java

```diff
@@ -73,11 +73,7 @@ public static void startMetastoreAndSpark() {
     SparkTestBase.catalog = (HiveCatalog)
         CatalogUtil.loadCatalog(HiveCatalog.class.getName(), "hive", ImmutableMap.of(), hiveConf);
 
-    try {
-      catalog.createNamespace(Namespace.of("default"));
-    } catch (AlreadyExistsException ignored) {
-      // the default namespace already exists. ignore the create error
-    }
+    createNamespace(catalog, Namespace.of("default"));
```

Comment:
This is really just a question: does it worth to create a twoliner method which is used only 2 places?

Reply:
In this case, it would be a good idea to have that method to ensure the namespace is created in the same manner i.e using Catalog methods.
I'm fine with removing the method if required.

Reply:
Your answer helped me to understand, why I felt ambiguous about this solution. We are in the same situation here than in the other places, like TestSparkReaderDeletes.java, and there we chose to inline the check.

Reply:
> We are in the same situation here than in the other places, like TestSparkReaderDeletes.java, and there we chose to inline the check.

`SparkTestBaseWithCatalog` extends `SparkTestBase`, thus the method can be reused in the child class. 
However, `TestSparkReaderDeletes` does not fall in the above inheritance hierarchy, thus we had to inline it. 

There also does not exist a TestUtil class for Spark tests or else we could have added the method to the util class and reused it everywhere.